### ISSUE TITLE
Missing biomes

### ISSUE THREAD

**Issue by ghost** [b3spectacled/modern-beta#2]
The following biomes are missing in the pack which causes soul sand beaches and terrain generating above Y 127:
```
taiga_hills
mountains
wooded_mountains
birch_forest_hills
wooded_hills
```
and probably more.

It seems like these biomes (and their alternates) generate regardless of the biomes set in overworld.json and regardless of the weirdness factor being set to 0.

**Comment by b3spectacled** [comment]
I think this is the result of the [previous issue](https://github.com/b3spectacled/modern-beta/issues/1), when the world generates biomes it isn't supposed to.  I'll fix the surface builder so it won't conflict with the vanilla one and generate those soul sand beaches whenever something like this does happen.

**Comment by tmewett** [comment]
Is this fixed by the namespace changes in 79000cbc8a8422fe6134c4f58cd5a802fa2b23c1 ?

**Comment by b3spectacled** [comment]
Yes, there should no longer be any conflicts with vanilla biomes/overworld.  Do open a new issue if you find a problem though.
